CMakeLists.txt

```diff
@@ -402,6 +402,35 @@ if (NOT MINGW AND NOT APPLE)
     set (CMAKE_EXE_LINKER_FLAGS "-Wl,-rpath=${_qt5Core_install_prefix}/lib")
     # install desktop file
     install( FILES build/mscore.desktop DESTINATION share/applications)
+    # compress man pages if gzip is present on system
+    set(MAN_TARGET ${PROJECT_SOURCE_DIR}/build/mscore.1)
+    find_program( GZIP_EXECUTABLE gzip DOC "A tool for compressing manpages (optional)." )
```

Comment:
CMAKE_SYSTEM_NAME matches "OpenBSD" is the correct way to do it, and the way we always do it on OpenBSD.

Not only is that FAQ page five years old, it is for something different. It is unacceptable for OpenBSD ports and packages to have compressed man pages.